srcpkgs/i3/template

```diff
@@ -14,7 +14,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="https://i3wm.org/"
 distfiles="${homepage}/downloads/${pkgname}-${version}.tar.xz"
-checksum=8dc5205e0f3443d58661e30783521fd9c6d426d1e7f3667ab12f2ac1b2d32078
+checksum=edfd781285c654a05dc6db272ce0300c4428e6fbd875d99964e57de25b41bc1e
 
 do_check() {
```

Comment:
make_check=no # comment

Reply:
sorry, don't understand why add this, can you explain to me?

Reply:
one more style nitpick, move $make_check to the first main block (after checksum), it's a variable and we put most of the variables to that block. xlint only checks the first block, so it won't warn you about this.

Reply:
right, it makes sense to put in there :)